src/libraries/Fuzzing/DotnetFuzzing/Program.cs

```diff
@@ -361,7 +384,14 @@ private static string GenerateLocalRunHelperScript(IFuzzer fuzzer)
     {
         string script = $"%~dp0/libfuzzer-dotnet.exe --target_path=%~dp0/DotnetFuzzing.exe --target_arg={fuzzer.Name}";
 
-        if (fuzzer.Dictionary is not null)
+        // We don't support dictionaries and corpora at the same time yet, and some fuzzers
+        // put corpus in dictionary to work around lack of corpus setup in OneFuzz. Locally
+        // use corpus as corpus if available as it is more effective that way.
+        if (fuzzer.Corpus is not null)
+        {
+            script += " %~dp0/corpus";
```

Comment:
Please move this after the "additional arguments" so that it becomes the last option.

Going by the text in https://llvm.org/docs/LibFuzzer.html#options
> To run the fuzzer, pass zero or more corpus directories as command line arguments. The fuzzer will read test inputs from each of these corpus directories, and any new test inputs that are generated will be written back to the first corpus directory

This way if you use a custom folder when fuzzing locally, you'll see the inputs being written there instead of in the deployment folder.